misoc/software/libbase/system.c

```diff
@@ -80,6 +96,8 @@ void flush_l2_cache(void)
 		__asm__ volatile("lw %0, (%1+0)\n":"=r"(dummy):"r"(addr));
 #elif defined (__or1k__)
 		__asm__ volatile("l.lwz %0, 0(%1)\n":"=r"(dummy):"r"(addr));
+#elif defined (__vexriscv__)
+		__asm__ volatile("lw x0, 0(%0)\n":"r"(addr));
```

Comment:
We could simply replace this chunk of assembly with a volatile load.

Reply:
I agree, but hadn't touch it as it looked like black magic ^^

Reply:
It's just touching enough code pages that everything else in cache gets evicted.